brainglobe_stitch/file_utils.py

```diff
@@ -363,3 +364,107 @@ def safe_find(root: ET.Element, query: str) -> ET.Element:
         raise ValueError(f"No element found for query {query}")
 
     return element
+
+
+def write_bdv_xml(
+    output_xml_path: Path,
+    input_xml_path: Path,
+    hdf5_path: Path,
+    image_size: Tuple[int, ...],
+    num_channels: int,
+) -> None:
+    """
+    Write a Big Data Viewer (BDV) XML file.
+
+    Parameters
+    ----------
+    output_xml_path: Path
+        The path to the output BDV XML file.
+    input_xml_path: Path
+        The path to the input BDV XML file.
+    hdf5_path:
+        The path to the output HDF5 file.
+    image_size:
+        The size of the image in pixels.
+    num_channels:
+        The number of channels in the image.
+    """
+    input_tree = ET.parse(input_xml_path)
+    input_root = input_tree.getroot()
+    base_path = safe_find(input_root, ".//BasePath")
+
+    root = ET.Element("SpimData", version="0.2")
+    root.append(base_path)
+
+    sequence_desc = ET.SubElement(root, "SequenceDescription")
+
+    image_loader = safe_find(input_root, ".//ImageLoader")
+    hdf5_path_node = safe_find(image_loader, ".//hdf5")
+    # Replace the hdf5 path with the new relative path
+    hdf5_path_node.text = str(hdf5_path.name)
+    sequence_desc.append(image_loader)
+
+    view_setup = safe_find(input_root, ".//ViewSetup")
+    # Replace the size of the image with the new size
+    view_setup[2].text = f"{image_size[2]} {image_size[1]} {image_size[0]}"
```

Comment:
just checking presumably this flipping of dims is intentional

Reply:
Yup! I'll add a comment to mark it as such.